packages/deploy-script-support/test/unitTests/test-install.js

```diff
@@ -9,10 +9,15 @@ import { resolve as importMetaResolve } from 'import-meta-resolve';
 
 import '../../exported.js';
 
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { makeScalarBigMapStore } from '@agoric/vat-data';
 import { makeInstall } from '../../src/install.js';
 
 test('install', async t => {
-  const { zoeService: zoe } = makeZoeKit(fakeVatAdmin);
+  const { zoeService: zoe } = makeZoeKit(
+    makeScalarBigMapStore('zoe baggage', { durable: true }),
+    fakeVatAdmin,
+  );
```

Comment:
this is common enough that there should be a single function to do it. perhaps `makeZoeKitForTest` in a `tools` export. Then most of these changes would be four chars at the call site (and an extra import, but on a package that's already a dependency).

Moreover, Zoe's baggage should always be durable, so the caller shouldn't be allowed to pass in any MapStore.

Reply:
I introduced a `makeZoeForTest`, `makeZoeKitForTest` and common `setupZoeForTest`, and refactored all the tests to use these new tools. Its was a big refactor, but I think it paid out!

Reply:
Looks it to me!